src/aap_eda/core/models/activation.py

```diff
@@ -74,6 +78,31 @@ class Meta:
     status_updated_at = models.DateTimeField(null=True)
     status_message = models.TextField(null=True, default=None)
 
+    def save(
+        self,
+        force_insert=False,
+        force_update=False,
+        using=None,
+        update_fields=None,
```

Comment:
I think we should not limit the api of the method overriding the params. Instead we should use `*args, **kwargs` and passing through them in the call to the parent.